### ISSUE TITLE
Proposal fix missingmethodexception

### ISSUE THREAD

**Issue by 304NotModified** [approvals/ApprovalTests.Net#167]
Hi,

I got also this exception: (just installed https://www.nuget.org/packages/ApprovalTests/)

```
Result Message: System.MissingMethodException : 
Method not found: 'System.Diagnostics.StackTrace ApprovalUtilities.CallStack.Caller.get_StackTrace()'.
```

Updating `ApprovalUtilities` fixed the issue. (as found on [stackoverflow](http://stackoverflow.com/questions/36908221/approval-test-throws-system-missingmethodexception)).

This is probably a backwards-compatiblity issue.

Isn't a good idea to update the minimal dependency of ApprovalUtilities in ApprovalTests package?  (e.g. ApprovalUtilities (>= 3.0.11)`

**Comment by 304NotModified** [comment]
Update, it looks like this is OK in the [.nuspec](https://github.com/approvals/ApprovalTests.Net/blob/master/ApprovalTests/ApprovalTests.nuspec), so maybe releasing a new release of ApprovalTests package?

**Comment by 304NotModified** [comment]
I see this is fixed in 3.0.13:

![image](https://user-images.githubusercontent.com/5808377/35798152-3bc404dc-0a62-11e8-9b46-17730ca8b2a8.png)
